packages/frontend/src/components/composer/Composer.tsx

```diff
@@ -535,11 +535,33 @@ const Composer = forwardRef<
           // (`id='chat-section-heading'`) is probably enough.
         }
       >
-        <div className='upper-bar'>
+        <section
+          // Keep in mind that this element also changes when switching
+          // between chats.
+          // We probably want such changes to be announced still.
```

Comment:
Hmm I don't think so. This is confusing and useless when switching chats, switching chats can be considered a form of "navigation", thus we must not announce the changes.

I would like to refactor `useDraft` such that it returns `loadingDraft`, which we can then use to only apply `aria-live` when we have finished loading the draft initially.
But there is another MR pending which deals which moves `useDraft` to a separate file. Let's do this later to avoid conflicts.